samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor

```diff
@@ -151,15 +153,35 @@
                 {
                     <td>
                         <input type="checkbox" checked=@node.Publishing @onchange="@((EventArgs) => SetPublishing(EndpointId, node, EventArgs))" />
-                        <a href="javascript: void(0)" @onclick="@(() => OpenDrawer(node))">
-                            @{string output = (node.Publishing == true && node.PublishedItem != null) ? $"On, sampling {node.PublishedItem?.SamplingInterval.Value.TotalMilliseconds} ms" : "Off";}
-                            @output
-                        </a>
+                        @{string output = (node.Publishing == true) ? "On" : "Off";}
+                        @output 
+                    </td>
+                    <td>
+                        @{_drawerType = drawer.Publisher;}
+                        @if (node.Publishing == true && node.PublishedItem != null)
+                        {
+                            <label>Sampling:</label><a href="javascript: void(0)" @onclick="@(() => OpenDrawer(node))">
+                                @(node.PublishedItem.SamplingInterval?.TotalMilliseconds.ToString() ?? "1000") ms<br />
+                            </a>
+                            <label>Publishing: </label><a href="javascript: void(0)" @onclick="@(() => OpenDrawer(node))">
+                                @(node.PublishedItem.PublishingInterval?.TotalMilliseconds.ToString() ?? "1000") ms<br />
+                            </a>
+                            <label>HeartBeat: </label><a href="jpavascript: void(0)" @onclick="@(() => OpenDrawer(node))">
+                                @(node.PublishedItem.HeartbeatInterval == null ? "-" : node.PublishedItem.HeartbeatInterval?.TotalMilliseconds.ToString() + " ms")
```

Comment:
I think the heartbeat is expressed in seconds, at least this is the convention from the old publisher

Reply:
you are right. I'll change it